prow/plugins/lgtm/lgtm.go

```diff
@@ -174,9 +174,9 @@ func handlePullRequestReview(gc githubClient, config *plugins.Configuration, own
 	// If we review with Approve, add lgtm if necessary.
 	// If we review with Request Changes, remove lgtm if necessary.
 	wantLGTM := false
-	if e.Review.State == "approve" {
+	if e.Review.State == github.ReviewStateApproved {
```

Comment:
@guineveresaenger I changed these strings to the new ReviewState constants added in https://github.com/kubernetes/test-infra/pull/8140/commits/d93750992ab0662674095650d3b0961191133f05. Please take a look to make sure I did it right!

Reply:
Were these previously wrong then? They are capitalized here now.

Reply:
Based on my testing, they were previously incorrect. I only ever saw capitalized and past tense strings (`APPROVED` vs `approve`) in [docs](https://developer.github.com/v3/pulls/reviews/#create-a-pull-request-review) and ad-hoc testing:

```shell
# errors elided
for i in (seq 65600 65645); do
  curl -s https://api.github.com/repos/kubernetes/kubernetes/pulls/$i/reviews\?access_token=blah | jq '.[].state'
done

"APPROVED"
"COMMENTED"
"APPROVED"
"COMMENTED"
"COMMENTED"
"APPROVED"
"CHANGES_REQUESTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"APPROVED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"APPROVED"
"APPROVED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
"COMMENTED"
```